pages/docker_compose.md

```diff
@@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows, your Docker daemon has only l
 
 </div>
 
+
```

Comment:
If it does only happen with JUML, then yes, by all means.

Le 20 juin 2016 09:58:38 GMT+02:00, Pierre Besson <notifications@github.com> a écrit :
>> @@ -61,6 +61,16 @@ If you are using Docker Machine on Mac or Windows,
>your Docker daemon has only l
>>  
>>  </div>
>>  
>> +
>
>I think we should put a title here like `Using JHipster UML in Docker`
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/jhipster/jhipster.github.io/pull/280/files/1dc7c082bbe76bc42f82323d2390d80ff0516c8d#r67648853

-- 
Mathieu ABOU-AICHI